### ISSUE TITLE
No Pilots ( Intel Still works)

### ISSUE THREAD

**Issue by EnchaJini** [3vi1/IMP#57]
No pilots on the Pilots tab.  intel still comes in and changes system statuses but there's no alerts since IMP can't see what character your on.

**Comment by blade-in-water** [comment]
im having the same problem ,, so imp doesnt know where i am :(

**Comment by GankEve** [comment]
Same issue with 5 of our corp , no  pilots and no location data, so warning not working but still tracking the intel channels.

**Comment by 3vi1** [comment]
See bug #55.  CCP has purposely crippled logging in the last patch.  The workaround will be to make it so that you can manually set your location in IMP, but it will be several days before I can motivate myself do it when CCP keeps making the game objectively worse.  Half the reason I started IMP was so that I/players didn't need to do that in Vintel.  :\

**Comment by EnchaJini** [comment]
Yeah i tried my previous Intel Program **(SpyGlass)** and it also reports intel on the Map like IMP does. but doesnt even do an alert sound. like it cant tell who your on. 

i Manually set my location on **(SpyGlass)** and i didnt get an alert either.  **(will test further and edit my statement if it does)**

most likely Vintel is having the same issues

**Comment by EnchaJini** [comment]
Just did a quick Test in my Alliances Intel Channel. and yeah no Alert from Spyglass. even though it knows where i am.

**Comment by Pocketfrog** [comment]
I wondered what was going on with the sound, i suppose we just have to not be lazy for a bit longer

**Comment by Draytek5** [comment]
@3vi1 - I note that N.E.A.R 2 is able to track pilot location via Eve's SSO and ESI. Is this a viable option for IMP?

**Comment by EnchaJini** [comment]
one of my Corpies.. tells me Vintel still is working.

**Comment by 3vi1** [comment]
I did not get to play/test last night, but I've gone ahead and built/uploaded a pre-release that fixes the character location:  https://github.com/3vi1/IMP/releases/tag/v0.9.7.7

Hopefully I'll get time tonight to test proper and wrap up any remaining issues.
